Generator/include/CGAL/point_generators_3.h

```diff
@@ -302,6 +305,168 @@ void Random_points_in_tetrahedron_3<P, Creator>::generate_point() {
 }
 
 
+template <class P, class Mesh>
+class Random_points_on_triangle_mesh_3 : public Generic_random_point_generator<
```

Comment:
You can add `VertexPointMap` as template parameter, with what you currently have as default